app/Listeners/QueueMessage.php

```diff
@@ -33,20 +33,18 @@ public function handle(QueueMessageRequest $event)
         $resources = $event->resources;
 
         // Build the email.
-        $email = new Email($resources['message'], $resources['contest'], $resources['competition'], $resources['users']);
-
-        foreach ($email->allMessages as $message) {
-            $content = $message['message'];
+        $email = new Email($resources);
```

Comment:
Gotcha. Good to know. Manager doesn't have a dedicated binding, so the FUUUUURNS approach is the one to go with 💃